Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -30,10 +35,19 @@ final class Analytics {
         tracker.exceptionAutotracking = false
         tracker.diagnosticAutotracking = false
     }
+
+    private static func getTestContext(from toggle: Unleash.Toggle.Name) -> SelfDescribingJson? {
+        let variant = Unleash.getVariant(toggle).name
+        guard variant != "disabled" else { return nil }
```

Comment:
Just out of curiosity, is the "nil" return a better option than error handling? In Android I would try hard not to return "null" at all in order to avoid explicit null checks and other related issues :tropical_drink:

Reply:
Yeah dealing with Null values is very opaque in Swift and you can see in the Type whether something is Optional (aka can be Null) That's why it is common practice to deal with Null values.